src/include/distributed/multi_router_executor.h

```diff
@@ -9,16 +9,26 @@
 #ifndef MULTI_ROUTER_EXECUTOR_H_
 #define MULTI_ROUTER_EXECUTOR_H_
 
+#include "libpq-fe.h"
```

Comment:
Whenever the include leads to including a significant amount of code into places that don't need the information. E.g. there's no need for multi_executor.c to know about libpq and such.